.github/workflows/pr_check_load_test.yml

```diff
@@ -34,13 +33,20 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+    
+    - uses: actions/checkout@v2
```

Comment:
This was removed earlier due to a security issue - https://github.com/Azure/load-testing/pull/65 Do we need it now or can we remove it?

Reply:
this is only ran on releases, and this is not removed from releases :(